Unix/configure

```diff
@@ -543,6 +543,7 @@ fi
 outputdir=$configuredir/$outputdirname
 
 if [ "$dev" = "1" ]; then
+  enable_preexec=1
```

Comment:
Why make this feature optional?  And why only for dev builds?

Reply:
preexec is enabled for --microsoft also. We only add it to dev